tests/modules/nf-core/bcftools/norm/test.yml

```diff
@@ -1,15 +1,19 @@
 - name: bcftools norm test_bcftools_norm_no_tbi
   command: nextflow run ./tests/modules/nf-core/bcftools/norm -entry test_bcftools_norm_no_tbi -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/bcftools/norm/nextflow.config
   tags:
-    - bcftools
     - bcftools/norm
+    - bcftools
   files:
     - path: output/bcftools/test2.vcf.gz
+      md5sum: a7524541481cd4e7d38267ded10253e2
+    - path: output/bcftools/versions.yml
 
 - name: bcftools norm test_bcftools_norm_tbi
   command: nextflow run ./tests/modules/nf-core/bcftools/norm -entry test_bcftools_norm_tbi -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/bcftools/norm/nextflow.config
   tags:
-    - bcftools
     - bcftools/norm
+    - bcftools
   files:
     - path: output/bcftools/test2.vcf.gz
+      md5sum: a7524541481cd4e7d38267ded10253e2
+    - path: output/bcftools/versions.yml
```

Comment:
can you add a test for at least one other output type

Reply:
Definitely! Thanks for the suggestion @FriederikeHanssen